src/styles.scss

```diff
@@ -14,6 +14,9 @@
  *    limitations under the License.
  */
 
+// TODO: Angular Material theme to be changed to a custom one
+@import '~@angular/material/prebuilt-themes/indigo-pink.css';
```

Comment:
Indigo-pink? 
A snack bar? 

This might be worthy of seeing in person before merging

Reply:
A material theme refers to 5 different colours: primary, accent, warning, background, foreground.  Currently the only two components added are snackbar and a button, neither of which use the pink accent and I have no intentions of using the accent colour anytime before we get a custom theme.  The primary colour indigo currently being used looks surprisingly similar to the rest of Dockstore.

Reply:
Ok, apparently I was wrong.  The button did not use the accent colour, but the snackbar used it for the "Dismiss" word.  It just wasn't apparent to me.
![selection_022](https://user-images.githubusercontent.com/24548904/39535384-c8c94fbc-4e01-11e8-9581-c85a2462ad88.png)